unison-src/transcripts/suffixes.output.md

```diff
@@ -38,7 +38,7 @@ The `view` and `display` commands also benefit from this:
 ```ucm
 .> view List.drop
 
-  builtin builtin.List.drop : Nat -> [a] -> [a]
+  builtin builtin.List.drop : builtin.Nat -> [a] -> [a]
```

Comment:
This could happen if a fully-qualified PPE is being used here for some reason - are all type names being shown fully qualified. 

It's also possible `Nat` is no longer unique in this transcript's codebase. You could dump out the names to see what's in scope.

Reply:
In the previous and subsequent stanzas, it shows up at `Nat`.  I'm not aware of anything in the branch that would've changed the PPE, but I guess I can dump it out eventually.

Reply:
Weird. Is it something specifically with printing out builtin types signatures?